app/templates/static-pages/destroy.hbs

```diff
@@ -1,9 +1,7 @@
 <p>Weet je zeker dat je deze informatie pagina wilt verwijderen?</p>
 
-<button {{action 'destroy'}} class='btn btn-danger' type='button'>Ja</button>
-<LinkTo @route='static-pages.show' @model={{model.id}} class='btn btn-default'>
-  Nee
-</LinkTo>
+<button {{action 'destroyModel'}} class='btn btn-danger' type='button'>Ja</button>
```

Comment:
On all other destroy confirmations (articles, groups, etc.), the options are "Verwijderen" and "Annuleren", but here it is "Ja" and "Nee". Not a major issue but it is inconsistent